Automattic/woocommerce-payments
# Add error notice when using ISK with decimals

Fixes #6232 

Ref: pNPgK-6sv-p2

#### Changes proposed in this Pull Request
Add error notice for the merchant when using ISK with decimals.

#### Screenshot
<img width="998" alt="Screen Shot 2023-05-05 at 1 18 33 PM" src="https://user-images.githubusercontent.com/56378160/236561709-2b0c764e-7d03-4e34-93c6-663453573dcd.png">


#### Testing instructions

1. Go to WC settings -> General
2. Set currency to Icelandic Króna and Number of decimals to 2
3. See notice `Unsupported currency: Icelandic Króna does not accept decimals. Please update your currency number of decimals to 0 or select a different currency. [Visit settings](http://merchant.local:10004/wp-admin/admin.php?page=wc-settings)`
4. Checkout as a shopper
5. Should not see `We’re not able to process this request. Please refresh the page and try again.` and able to checkout.
6. Go back to the settings and set Number of decimals to 0
7. Should not see the notice

*

-------------------


**Post merge**
